wiki/Guides/Compressing_Files/en.md

```diff
@@ -15,59 +17,19 @@ The `.osz` package is actually a `.zip` file, and will compress your beatmap, al
 Video
```

Comment:
Switch to atx (`#`) on remaining heading level 2's.

Reply:
there's a lot to fix about this file , opened an issue instead of doing so in the same PR